### PR TITLE
Phase2-gem61 Try to update some of the region definitions for calorimeter and muon systems

### PR DESCRIPTION
**Pull request by bsunanda**
#### PR description:

Try to update some of the region definitions for calorimeter and muon systems

#### PR validation:

Use the runTheMatrix test workflows

#### if this PR is a backport please specify the original PR and why you need to backport that PR:

Nothing special

### CODE REVIEW

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by cvuosalo** [comment]
+1

File: Geometry/MuonSimData/data/muonProdCuts/2026/v1/muonProdCuts.xml [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+  </SpecPar>
+  <SpecPar name="MuonSensitive_DT-CSC">
+   <!-- 1 keV for gamma/e-, 1 MeV for e+ -->
+   <PartSelector path="//MB.*PhiGas"/>
+   <PartSelector path="//MB.*ZGas"/>
+   <PartSelector path="//ME.*ActiveGasVol"/>
+   <Parameter name="CMSCutsRegion" value="MuonSensitive_DT-CSC" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="25.*mm"/>
+   <Parameter name="ProdCutsForElectrons" value="1.*mm"/>
+   <Parameter name="ProdCutsForPositrons" value="2.5*m"/>
```
Comment (civanch) [inline Geometry/MuonSimData/data/muonProdCuts/2026/v1/muonProdCuts.xml:51]: max value should be 1 cm

File: Geometry/MuonSimData/data/muonProdCuts/2026/v1/muonProdCuts.xml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+   <!--    <PartSelector path="//REA.*"/>  -->
+   <PartSelector path="//REB.*"/>
+   <PartSelector path="//REC.*"/>
+   <!--    <PartSelector path="//RED.*"/>  -->
+   <PartSelector path="//REE.*"/>
+   <PartSelector path="//REF.*"/>
+   <PartSelector path="//REG.*"/>
+   <PartSelector path="//REH.*"/>
+   <Parameter name="CMSCutsRegion" value="MuonSensitive_RPC" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="8.5*m"/>
```
Comment (civanch) [inline Geometry/MuonSimData/data/muonProdCuts/2026/v1/muonProdCuts.xml:65]: 1 cm

File: Geometry/MuonSimData/data/muonProdCuts/2026/v2/muonProdCuts.xml [hunk 3]
```diff
@@ -0,0 +1,71 @@
+<?xml version="1.0"?>
+<DDDefinition>
+ <SpecParSection label="ProductionCuts" eval="true">
+  <SpecPar name="Muon">
+   <PartSelector path="//MUON"/>
+   <Parameter name="CMSCutsRegion" value="Muon" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="100.*mm"/>
```
Comment (civanch) [inline Geometry/MuonSimData/data/muonProdCuts/2026/v2/muonProdCuts.xml:7]: 1 cm

File: Geometry/MuonSimData/data/muonProdCuts/2026/v1/muonProdCuts.xml [hunk 4]
```diff
@@ -0,0 +1,70 @@
...
+   <PartSelector path="//YN.*"/>
+   <PartSelector path="//YE1.*"/>
+   <PartSelector path="//YE2.*"/>
+   <PartSelector path="//YE3.*"/>
+   <!-- if you want to re-stage YE4, then remove the comments around this element   <PartSelector path="//YE4.*"/>   -->
+   <PartSelector path="//ShieldingME4.*"/>
+   <PartSelector path="//ShieldingBoronPoly.*"/>
+   <PartSelector path="//ME11SupportDisk"/>
+   <Parameter name="CMSCutsRegion" value="MuonIron" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.*mm"/>
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Geometry/MuonSimData/data/muonProdCuts/2026/v1/muonProdCuts.xml:28]: I want to move phase2/muonProdcuts.xml with its default cuts to muonProdcuts/2026/v1 and later replace them in the workflows. This is to keep the old record. I shall change the values for v2

File: Geometry/MuonSimData/data/muonProdCuts/2026/v2/muonProdCuts.xml [hunk 5]
```diff
@@ -0,0 +1,71 @@
...
+   <PartSelector path="//YN.*"/>
+   <PartSelector path="//YE1.*"/>
+   <PartSelector path="//YE2.*"/>
+   <PartSelector path="//YE3.*"/>
+   <!-- if you want to re-stage YE4, then remove the comments around this element   <PartSelector path="//YE4.*"/>   -->
+   <PartSelector path="//ShieldingME4.*"/>
+   <PartSelector path="//ShieldingBoronPoly.*"/>
+   <PartSelector path="//ME11SupportDisk"/>
+   <Parameter name="CMSCutsRegion" value="MuonIron" eval="false"/>
+   <Parameter name="ProdCutsForGamma" value="1.*mm"/>
```
[the remark this answers is not in the document]
Comment (bsunanda) [reply Geometry/MuonSimData/data/muonProdCuts/2026/v2/muonProdCuts.xml:28]: @civanch Please check the values set in 2026/v2/muonProdcuts.xml. I do not want to modify the older values in v1 which will be kept as record

**Comment by bsunanda** [comment]
@cmsbuild Please test

**Comment by civanch** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**